_guide/_pages/interviews/core-values.md

```diff
@@ -54,108 +54,61 @@ Begin by introducing yourself by saying this or something similar to it:
 
-Make a note of key phrases or words that the candidate mentions their response (e.g., “I want my work to have a meaningful impact”); you can use these for hooks that there are hooks for questions later in the interview as you transition from one set of questions to the next.
+Make a note of key phrases or words that the candidate mentions their response (e.g., “I want my work to have a meaningful impact”); you can use these hooks for questions later in the interview as you transition from one set of questions to the next.
 
 ### Relevant work experience
 
 After the introduction, follow up with a discussion around relevant work experience. Tying into anything the candidate might have already responded with as well as their resume, guide the interview toward specific work experience(s) that are of interest to 18F’s current and future needs.
 
 This question could go in a few different directions depending on the candidates experience. Ask one (or more) of the following:
 
-- [Tell me a bit more about your experience at COMPANY / on PROJECT](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
+- [Tell me a bit more about your experience at COMPANY / on PROJECT](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.8gbmhtmnymv9)
     - What challenges did you face and how did you overcome them?
     - If a project: Why did the project succeed/fail?
     - If a company: Why did/are you ultimately decide/deciding to move on?
 
-- [Have you worked directly with customers/clients before?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
+- [Have you worked directly with customers/clients before?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.8w4325mhjtnd)
     - If applicable: Could you speak about a time when you dealt with a particularly difficult client?
 
-- [Talk to me about how you prioritize features.](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
-    - How do you balance requests from various stakeholders, especially when their priority is different than yours?
+- [Tell me about your experience working on teams that directly involved non-engineers. How did you approach working with others with different professional skills?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.6qssa3tfp2ht)
+    - How did you resolve any conflicts that arose?
+    - How did you incorporate other perspectives into your technical choices, especially when that might lead you to a different technical strategy?
 
-- [Have you had a chance to participate in user-centered design and/or user research?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.z49dyqcuxle5)
```

Comment:
I'd keep this one. How they answer it speaks to their willingness to embrace research, but also how human-centered they are as an engineer. Perhaps broaden it a bit to "how do you incorporate user-centerered design and research into engineering" with a follow up about if you've actually participated